pwiz_tools/Skyline/Model/Databinding/Entities/PrecursorResult.cs

```diff
@@ -116,6 +116,8 @@ public double? RatioDotProduct {
         public double? AverageMassErrorPPM { get { return ChromInfo.MassError; } }
         [Format(NullValue = TextUtil.EXCEL_NA)]
         public int? CountTruncated { get { return ChromInfo.Truncated; } }
+        [Format(Formats.STANDARD_RATIO, NullValue = TextUtil.EXCEL_NA)]

```

Comment:
We might want to use `Formats.Percent` here instead of `Formats.STANDARD_RATIO`.
(There is also `Formats.PEAK_AREA_NORMALIZED` which is a percent with up to four decimal places)